doc/nrf/ug_bt_qualification.rst

```diff
@@ -0,0 +1,109 @@
+.. _ug_bt_qualification:
+
+Bluetooth qualification
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Every product using the Bluetooth® LE technology must be certified (qualified) by Bluetooth Special Interest Group (SIG).
+This condition must be met to perform the following actions:
+
+* Demonstrate that your products satisfy the requirements of the Bluetooth license agreements and specifications.
+* Use Bluetooth trademark and logos.
+* Sell products that use Bluetooth LE technology.
+
+As a result, your products will be listed and can be promoted on the official Bluetooth SIG page.
+
+
+Pre-qualified design
+********************
+
+By using pre-qualified designs, you can omit the testing phase during your product's qualification.
+
+The following product types are used and qualified by Nordic Semiconductor:
+
+* End product
+* Controller Subsystem product
+* Host Subsystem product
+* Profile Subsystem product
+* Component product
+
+All versions of Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems and Components are created and qualified by Nordic Semiconductor by default.
+Some Profile Subsystems (e.g., :ref:`ug_bt_mesh` versions) are qualified as well.
+If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_ yet, run the tests through the `Profile Tuning Suite (PTS)`_ tool and qualify the needed product by yourself.
+
+
+Qualification process
+*********************
+
+The product qualification cannot be inherited, hence you must complete this process by yourself.
+You can either perform a full qualification process for a new product, or edit an existing entry for a product that has already been qualified.
+
+New entry creation
+==================
+
+The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
+To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.
+
+Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following paths:
```

Comment:
It looks like this is required for both new entry AND existing entry. So maybe move this to a new preceding subsection "Qualification requirements"? You could mention there that the Launch Studio is this and that and that it is required, and in "New entry creation" and "Existing entry update" have just content that refers to the Launch Studio...

Reply:
Thank you, I see your point :) Let me explain why I wrote it this way. I wanted to keep this document as short and sweet as possible to avoid duplicating basic info from the SIG site. I think that end users will check SIG site first, and then will try to find Nordic-related details on Nordic page. So, yes, some parts are common for both cases, but I wanted to clearly separate new from existing entries sections so that end users can pick up and read only the most crucial parts for them - they know the basics already. If you remain unconvinced, we can discuss it via Teams ;)

Reply:
I became convinced.